dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESClient.java

```diff
@@ -76,12 +72,19 @@ private void initNode () {
                     shutDownNode();
 
                     String node_id = ConfigUtils.getServerId();
-                    _nodeInstance = nodeBuilder().
-                            settings(
-                                    ImmutableSettings.settingsBuilder().
-                                            put( "name", node_id ).
-                                            put( "script.native.related.type", RelationshipSortOrderScriptFactory.class.getCanonicalName() ).build()
-                            ).build().start();
+                    try{
+                        _nodeInstance = new Node(
+                            Settings.builder().
+                                put( "cluster.name", Config.getStringProperty("es.cluster.name")).
+                                put( "node.name", node_id ).
+                                put("path.home", "WEB-INF/elastic_search").
```

Comment:
what is this for? `put("path.home", "WEB-INF/elastic_search").`

Reply:
This path defines the location of the elastic search modules (those that we'll be using temporarily for transport client implementation) and plugins (so far, we have one plugin to support relationships sort order)

Reply:
I'll read this path from our properties files to avoid hard-coding it